### DISCUSSION TITLE
Miniplayer?

### DISCUSSION THREAD

**Discussion by Gumbyohson** [pear-devs/pear-desktop#668] [Q&A] [answered]
I cant seem to get the miniplayer to work. I usually like to keep the youtube app open in a small corner of the screen so i can quickly control it during work or downvote a song I dont like. If I resize the player to the size I need the thumbs icons are missing. Any ideas?

**Comment by Araxeus** [comment, marked as answer]
This is a native feature of youtube music, but I've added an option to disable it in #673 (see screenshot there)

From what I understand the issue was that the like buttons disappear so I've added an option to force show them (until they have no more space left)

Hope this will help 🙂

**Reply by Gumbyohson** [reply]
Do you know when this will be pushed to the releases for install or is this dynamic?

**Reply by Araxeus** [reply]
The PR needs to be merged to the main branch and then a new version needs to be released

Only the project owner can do that (@th-ch)

(Releases are currently not automated)
